websites/R/Replit/dist/metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "author": {
+    "name": "Ashutosh",
+    "id": "316943845596200960"
+  },
+  "service": "Replit",
+  "description": {
+    "en": "Replit.com is a San Francisco based start-up and an online IDE (integrated development environment) supporting 40 programming languages, including Python, Lua, JavaScript, Julia, C++, C++11, C, C#, HTML/CSS/JS, Python with turtle and LOLCODE. Repl.it allows users to write code and build apps using a browser, without having to install any software on their devices."
```

More fitting description could be the likes of such:
```suggestion
    "en": "Replit, formerly Repl.it, is a San Francisco based start-up and an online IDE (integrated development environment) supporting 40 programming languages, including Python, Lua, JavaScript, Julia, C++, C++11, C, C#, HTML/CSS/JS, Python with turtle and LOLCODE. Replit allows users to write code and build apps using a browser, without having to install any software on their devices."
```